### PR TITLE
B 19832 add weight stored to APIs

### PR DESCRIPTION
**Pull request by r-mettler**
## Summary
This PR adds a new column into the moving_expenses table called weight_stored and updates the moving-expenses API calls in Internal and GHC to use the weight_stored field. This is the first in a series of stories that will give Customers the ability to add amount of weight stored for a moving expense on their PPM shipment and allow Office users to see the amount on the Review Documents page. 

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login as a Customer and create and submit a new PPM shipment
2. Login as a Service Counselor and approve the PPM shipment
3. Login as the Customer and click the Upload PPM Documents button
4. Enter the data on the first page click Save & Continue
5. Add a new moving expense using type of "Storage"
6. Inspect in the Chrome dev tools and look at the Network tab and mto_shipments
7. Expand the shipment and look for moving expenses and expand your moving expense
8. You should see a ppm shipment id, the moving expense id (listed as just "id") and the eTag
9. Copy these values off to use in swagger calls
10. Open [swagger for Internal ](http://milmovelocal:3000/swagger-ui/internal.html#/ppm/updateMovingExpense)
11. Enter your copied values for the updateMovingExpense PATCH call
12. Enter weightStored in the body, something like this:   "weightStored": 3000
13. Update the "amount" to be greater than 1
14. Execute the call and verify that you get a success message and see your weightStored value
15. Optionally also look for your moving expense in the moving_expense database table to verify you have a weightStored value
16. Go back to your Customer page and pull the new eTag value
17. Open the [GHC swagger page](http://officelocal:3000/swagger-ui/ghc.html#/ppm/updateMovingExpense)
18. Enter your values for the PATCH call and a different weightStored amount to the body:   "weightStored": 4000
19. Execute and verify success and check DB table
20. Go to the [swagger for Internal ](http://milmovelocal:3000/swagger-ui/internal.html#/ppm/updateMovingExpense)
21. Scroll to the POST call to create a moving expense
22. Enter your ppm shipment id and execute
23. Verify the create was successful

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

**Review by antgmann — APPROVED** [review]
Code matches INT, no warnings or failures, migration is included in excel document, everything looks good.

**Review by CoryKleinjanCACI — APPROVED** [review]
Code matches INT. Everything looks good.

File: swagger/internal.yaml [hunk 1]
```diff
@@ -3534,9 +3540,8 @@ paths:
         Searches for an application parameter by name and value, returns nil if
         not found
       description: >-
-        The system will provide a parameter name given to them and if the name
-        provided is present in the database, then it will return the value
-        associated with that name
+        Searches for an application parameter by name and value, returns nil if
+        not found
       operationId: validate
```
Comment (cameroncaci) [inline swagger/internal.yaml:3545]: Tracking that this is a change originating from main's gen. Commit hashes look good, this code diff between the int branch can be ignored

**Review by cameroncaci — APPROVED** [review]

File: swagger/internal.yaml (hunk 1 above)
Comment (cameroncaci) [reply swagger/internal.yaml:3545]: image
Proof that it happens on my end

**Pull request merged**